css/signup.css

```diff
@@ -17,91 +18,101 @@ body {
   font-weight: 600;
   text-transform: capitalize;
 }
-.signUp {
-  padding:0rem 0.2rem;
+/*Wrapper*/
+#wrapper {
+  display: flex;
+  justify-content: center;
+  margin-top: 35px;
+}
+/**SignUpInfo Styles*/
+#signUpInfo {
+  background-color: #1f59bb;
+  color: #ffffff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#ffffff" to be "#fff" (color-hex-length)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5662357)